### DISCUSSION TITLE
IPv6 REST Call

### DISCUSSION THREAD

**Discussion by bernhard-teacher** [Budibase/budibase#8856] [Help]
Hi all,
my BudiBase Instance is selfhosted on DigitalOcean.

i want to run an Automation with an REST Datasource. This API is IPv6-Only and i get an error
{"code":"ENOTFOUND","type":"system"}

I think, this is because of the IPv6-Only Config of the API, with an IPv4-Call it works (i assigned temporarly an IPv4-Address).

From the bash on the server i can reach the API via IPv6.
From the container bbproxy i can't connect to the API (tested with curl).

Is there a workaround to enable IPv6-Calls from the containers?

Thanks!

**Comment by melohagan** [comment]
Hey @bernhard-teacher 

What format is the URL you are using in the REST Datasource? Are you using square brackets? https://stackoverflow.com/questions/64866276/node-fetch-with-ipv6-proxy

If that doesn't help, you may want to raise a bug report.

**Reply by melohagan** [reply]
Does this help at all? https://docs.budibase.com/docs/ipv6
